features/step_definitions/helper_services.rb

```diff
@@ -528,8 +528,9 @@
     logger.info "found existing iSCSI pod, skipping config"
   elsif _pod.exists?(user: admin, quiet: true)
     logger.warn "broken iSCSI pod, will try to recreate keeping other config"
-    @result = admin.cli_exec(:delete, n: _project.name, object_type: "pod", object_name_or_id: _pod.name)
-    raise "could not delete broken iSCSI pod" unless @result[:success]
+    step %Q{admin ensures _pod.name pod is deleted from the _project.name project}
```

Comment:
```suggestion
    step %Q{admin ensures #{_pod.name} pod is deleted from the #{_project.name} project}
```

Reply:
Thanks @akostadinov, I tried this way in my last commit but seems not work, I will have another try. 
My local cucushift env is totally broken so I pushed my new code and tried with Jenkins jobs, sorry for the noise.

Reply:
@duanwei33 , the point was to wrap ruby expressions in `#{...}`. I don't know if there are typos in the step name or missing quotes or something. In this case I'm pretty sure there need to be quotes around pod name and project. Something like

```suggestion
    step %Q{admin ensures "#{_pod.name}" pod is deleted from the "#{_project.name}" project}
```

I don't have an environment to test that though. I'm sure there are examples in the existing other steps where you can see how it was done. Or you can `grep -r ensure_deleted features/step_definitions` to see how to programatically create an object and delete it without calling into a step. Calling other steps from steps is discouraged in latest cucumber. But it is still fine I think.

Reply:
Hi @akostadinov, it's really embarrassing that I used {#_pod.name} but not #{_pod.name}, it works. But it's nice to talk to you here again. Thanks again for your help.